docker/hive/start-hive.sh

```diff
@@ -6,9 +6,13 @@ echo "Start Hive"
 METASTORE_PORT=${METASTORE_PORT:=9083}
 JMX_PORT=${JMX_PORT:=8008}
 JMX_PROMETHEUS_PORT=${JMX_PROMETHEUS_PORT:=8080}
+PROFILE=$1
 
+if [[ $PROFILE -eq "metastore" ]]
```

Comment:
This is a breaking change. We want to be able to run the metastore and server separately, and if there's no parameter - run both.